src/components/organisms/pages/page_school/School.jsx

```diff
@@ -0,0 +1,46 @@
+// import { Button, Icon } from "/src/components/Organisms/pages/index.js";
+
+import { Button } from "/src/components/molecules/Button";
+import { Icon } from "/src/components/molecules/Icon";
+import { FiLock } from "react-icons/fi";
+import image from "/src/assets/teacher.jpg";
+
+export const School = ({ showCardShadow }) => {
+  const shadowClasses = showCardShadow
+    ? "lg:shadow-lg lg:rounded-3xl lg:p-16"
+    : "";
+  return (
+    <div
+      className={`flex w-[310px] flex-col items-center lg:mx-16 lg:h-[770px] lg:w-full ${shadowClasses}`}
+    >
+      <h1 className="mb-6 text-3xl font-bold">Class books</h1>
+
+      <div className="flex flex-wrap justify-center gap-6 overflow-y-scroll lg:w-[800px]">
```

Comment:
You could try to see the possibility of editing the scroll visually, if not possible discuss it with me.

Reply:
I changed it with a tailwind plugin, you'll probably need a npm install after this merge, and it looks like this:
![scrollbar](https://github.com/javonunezz/teacher_book_front/assets/62600949/ff84a3b8-1870-46ce-aefe-93a096488e2b)